src/components/Scroll/Items/Item.tsx

```diff
@@ -60,14 +61,23 @@ const ScrollItem: React.FC<ScrollItemProps> = ({
     total: itemCount,
   };
 
+  console.log({ figure });
```

Comment:
@mathewjordan do you want to leave this here? Just checking, PR looks great! 👍

Reply:
Nope, thanks 🙏!